src/Digdir.Domain.Dialogporten.WebApi/Endpoints/V1/ServiceOwner/Dialog/ListDialogEndpoint.cs

```diff
@@ -30,3 +36,38 @@ public override async Task HandleAsync(ListDialogQuery req, CancellationToken ct
             validationError => this.BadRequestAsync(validationError, ct));
     }
 }
+
+public sealed class ListDialogEndpointSummary : Summary<ListDialogEndpoint, ListDialogQuery>
+{
+    public ListDialogEndpointSummary()
+    {
+        Summary = "Gets a list of dialogs";
+        Description = """
+                Performs a search for dialogs, returning a paginated list of dialogs. For more information see the documentation (link TBD).
+
+                * All date parameters must be parseable as UTC dates. Example: 2023-10-27T10:00:00Z
+                * See "continuationToken" in the response for how to get the next page of results.
+                * hasNextPage will be set to true if there are more items to get.
+                """;
+        Responses[200] = "Successfully returned the dialog list";
+        Responses[401] = Constants.SummaryErrorServiceOwner401;
+
+        RequestParam(p => p.ServiceResource, "Filter by one or more service resources");
+        RequestParam(p => p.Party, "Filter by one or more owning parties");
+        RequestParam(p => p.ExtendedStatus, "Filter by one or more extended statuses");
+        RequestParam(p => p.Status, "Filter by status");
+        RequestParam(p => p.CreatedAfter, "Only return dialogs created after this date");
+        RequestParam(p => p.CreatedBefore, "Only return dialogs created before this date");
+        RequestParam(p => p.UpdatedAfter, "Only return dialogs updated after this date");
+        RequestParam(p => p.UpdatedBefore, "Only return dialogs updated before this date");
+        RequestParam(p => p.DueAfter, "Only return dialogs with due date after this date");
+        RequestParam(p => p.DueBefore, "Only return dialogs with due date before this date");
+        RequestParam(p => p.VisibleAfter, "Only return dialogs with visible-from date after this date");
+        RequestParam(p => p.VisibleBefore, "Only return dialogs with visible-from date before this date");
+        RequestParam(p => p.Search, "Search string for free text search. Will attempt to fuzzily match in all free text fields in the aggregate");
+        RequestParam(p => p.SearchCultureCode, "Limit free text search to texts with this culture code, e.g. \"nb-NO\". Default: search all culture codes");
+        RequestParam(p => p.OrderBy, "Order by one or more fields with ascending or descending direction. Example: dueAt_asc,updatedAt_desc");
```

Comment:
Noen ideer om hvordan vi eksponerer gyldige order by verdier i swagger automagisk?

Reply:
Det er jo mulig å generere en liste over gyldige verdier her med reflection, men  ser ingen muligheter med NSwag for å oppgi den. Antagelig må parameteret knyttes til et schema i swaggeren, men tror det vil koste mer enn det vil smake å få til i kode. Man har alltids post-prosessering av hele swaggeren, men det vil vi nok helst unngå.